### DISCUSSION TITLE
Missing translations

### DISCUSSION THREAD

**Discussion by NicolasPetton** [NicolasPetton/Projectify#44] [General]
Hi @HueLiu, @joebordes, @gregorfischer!

Thank you all very much for contributing translations to Projectify!

As Projectify gets new features, it would be great if the translation packages were kept up-to-date. Would it be ok if I posted in this thread the new strings that need to be translated prior to publishing a release?

Thanks!
Nico

**Comment by joebordes** [comment]
Absolutely! I will try to help as much as I can.

**Comment by gregorfischer** [comment]
Of course. I'd be happy to help!

**Comment by NicolasPetton** [comment]
Awesome, thanks!

@HueLiu, @joebordes, @gregorfischer,
There's one new string that needs translating in `index.multids`: `ShowCompletedTodosByDefault`.
